lucien-ios/Subclasses/BottomBorderTextField.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BottomBorderTextField.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/12/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+final class BottomBorderTextField: UITextField {
+    private let bottomBorder = CALayer()
+
+    var borderColor: UIColor? {
+        get {
+            if let color = bottomBorder.backgroundColor {
+                return UIColor(cgColor: color)
+            } else {
+                return nil
+            }
+        }
+        set {
+            bottomBorder.backgroundColor = newValue?.cgColor
+        }
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createBottomBorder()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        createBottomBorder()
+    }
+
+    func createBottomBorder() {
```

Comment:
this can be private, others don't need it since you're setting it up at initialization.